Thorium-Sim/thorium-nova
# Electron now recognizes the client as the host

## Description

Generates a secret that is shared between the Electron shell, the HTTP server, and the web app which allows the server to know whether the connected client is coming from the Electron app that is hosting the HTTP server.

## Related Issue








Closes #124 

## How do you know the changes work correctly?






Manual Testing
- Build the app using `npm run build`
- Package the app using `npm run package`
- Open the Electron app for your platform inside the `/packages` folder
- You should see "You are the host" on the welcome page.

## Screenshots (if appropriate):

<img width="1136" alt="Screen Shot 2022-04-20 at 8 13 21 AM" src="https://user-images.githubusercontent.com/6558157/164227931-64f5bd13-3e84-4063-b9b4-20a95aa33e01.png">

## Types of changes



      change)

## Checklist:
